ci-operator/step-registry/baremetalds/devscripts/conf/serial/baremetalds-devscripts-conf-serial-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds devscripts conf command for serial suite ************"
+
+echo "export NUM_EXTRA_WORKERS=1" >> "${SHARED_DIR}/dev-scripts-additional-config"
+echo "export APPLY_EXTRA_WORKERS=true" >> "${SHARED_DIR}/dev-scripts-additional-config"
```

Comment:
We should really get rid of such conf step, and use `DEVSCRIPTS_CONFIG` instead defined directly in the workflow

Reply:
IINW that's because the configuration is currently defined at the job level, while it should stay at the workflow level. Probably that's another reason to have dedicated workflows per configuration (it could be addressed into another PR)